src/App.css

```diff
@@ -1,32 +1,270 @@
-  align-self: flex-start;
-  gap: 10px;
+  gap: 30px;
+  font-family: 'Montserrat', sans-serif;
+  align-items: center;
+  justify-content: space-between;
+  width: 88%;
+  margin: 0;
+  padding: 1.438rem 1rem 1.688rem 5rem;
+  background-color: var(--white-two);
+}
+
+.firstpart {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
 
-li {
-  list-style: none;
+.navlink {
+  text-decoration: none;
+  color: #121212;
+  opacity: 0.5;
 }
 
-.flexRow {
+.BOOKS,
+.categories {
+  margin-right: 50px;
+  text-transform: uppercase;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+}
+
+.Line {
+  width: 98%;
+  margin: 2.5rem 4.063rem 1.813rem 0;
+  border: solid 1px #e8e8e8;
+}
+
+.Title {
+  width: 10.875rem;
+  height: 1.5rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+  margin-bottom: 2rem;
+}
+
+/*  book style starts here */
+.container {
+  width: 80%;
+  margin: 1.313rem 5rem 0;
+  font-family: 'RobotoSlab', sans-serif;
+  background: #fff;
   display: flex;
+  justify-content: space-between;
   align-items: center;
-  gap: 10px;
+  padding: 2rem 4.188rem 1.625rem 1.688rem;
+  border: 1px solid #e8e8e8;
+}
+
+.main-wrapper {
+  padding-top: 1rem;
+  background-color: rgba(0, 0, 0, 0.05);
 }
 
-.flexColumn {
+.book {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.statistics {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.oval {
+  width: 5.625rem;
+  height: 5.625rem;
+  padding: 0.63rem 0.625rem 0.75rem 0.75rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.circular_oval {
+  position: absolute;
+  border-radius: 50%;
+  width: 4.25rem;
+  height: 4.25rem;
+  border: 5px solid #379cf6;
+  border-left-color: #e8e8e8;
+  -webkit-transform: rotate(45deg);
+  transform: rotate(45deg);
+}
+
+.bookdetails {
   display: flex;
   flex-direction: column;
+  justify-content: flex-start;
+  width: 300px;
+  font-family: 'Montserrat', sans-serif;
 }
 
-form {
+.category {
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.title {
+  color: rgb(0, 0, 0);
+  margin: 0;
+}
+
+.author {
+  color: #4386bf;
+  opacity: 0.7;
+  margin: 0;
+}
+
+.button {
+  background: transparent;
+  border: none;
+  padding-left: 0;
+  color: #4386bf;
+  font-size: 0.875rem;
+  font-weight: 300;
+  opacity: 0.7;
+}
+
+.actions {
+  margin-top: 1rem;
+}
+
+.completion {
+  display: flex;
+  flex-direction: column;
+}
+
+.completed {
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.divider {
+  height: 4.5rem;
+  margin: 0.75rem 1.063rem 0 0.938rem;
+  border: solid 1px #e8e8e8;
+}
+
+.chapteerprogress {
+  color: #121212;
+}
+
+.Current-Chapter {
```

Comment:
- [ ] I kindly suggest that you remove the width property on the .Current-Chapter because it's causing your `CURRENT CHAPTER` text to not be on One line as requested by the previous reviewer.

<img width="946" alt="image" src="https://user-images.githubusercontent.com/52934128/224483994-aa6cd90c-4f6f-49c7-b8c2-84a0ce2ebff9.png">

Reply:
thank you, done.

Reply:
@topeogunleye , i have removed it.

Reply:
@hakichris you're welcome